frontend/src/components/Repository.vue

```diff
@@ -327,13 +327,13 @@ export default {
   computed: {
     ...mapState({
       gem: state => state.gems.gem,
+      gems: state => state.gems.gems,
```

Comment:
I feel `gems/gemList` is more understandable than `gems/gems` but I'm fine to keep it as is.